Dockerfile

```diff
@@ -3,11 +3,7 @@
 # production: runs the actual app
 
 # Build builder image
-FROM ruby:3.2.2-alpine as builder
-
-# RUN apk -U upgrade && \
-#     apk add --update --no-cache gcc git libc6-compat libc-dev make nodejs \
-#     postgresql13-dev yarn
+FROM ruby:3.4.4-alpine AS builder
```

Comment:
It would be better to hardcode the alpine version, otherwise we will get updated automatically which would most likely cause issues
e.g. 3.4.4-alpine3.22
same for L56